modules/ossm-kiali-ossmc-plugin-install-cli.adoc

```diff
@@ -6,14 +6,14 @@
 [id="ossm-kiali-ossmc-plugin-install-cli_{context}"]
 = Installing OpenShift Service Mesh Console plugin using the CLI
 
-You can install the {SMProductName} (OSSMC) plugin using the CLI, instead of the {product-title} web console.
+You can install the {SMPlugin} using the CLI, instead of the {product-title} web console.
 
 .Prerequisites
 
-* OpenShift 4.12+ is installed.
-* Kiali Operator provided by Red Hat 1.73 is installed.
-* {SMProductName} (OSSMC) is installed.
-* Kiali Server is installed.
+* {product-title} is installed.
+* {KialiProduct} 1.73 or later is installed.
```

Comment:
For SMCP 2.x, the Kiali version will always be 1.73, so I do not understand the `or later` here. I prefer to remove it to avoid confusion for the users.

Reply:
@jshaughn , thoughts?

Reply:
Well, it's not true that for SMCP 2.x that Kiali 1.73 will always be installed.  For SMCP 2.3 and 2.4 the version would be 1.57 and 1.65, respectively.  **BUT**, for SMCP 2.5 or later, as required in the last bullet, the Kiali version will be 1.73.   And so I think it is safe to apply Fernando's recommended change.

Reply:
Yes, I meant 2.x that supports OSSMC, that is 2.5+. Thanks for the clarification Jay.